src/utils.ts

```diff
@@ -1,32 +1,105 @@
 import * as core from '@actions/core'
+import {execSync} from 'child_process'
 import moment from 'moment'
 import string from 'string-sanitizer'
 import YAML from 'yaml'
-
-export default function GetDateFormatted(date: Date): string {
+import {promisify} from 'util'
+import fs from 'fs'
+export function GetDateFormatted(date: Date): string {
   return moment(date).format('YYYYMMDD_HHmm')
 }
 
-export function parseYAML(filePath: string, repo: string | undefined, content: string): any {
+export function parseYAML(
+  filePath: string,
+  repo: string | undefined,
+  content: string
+): any {
   const defaultValue = 'Undefined' // Default value when json field is not defined
   let name = defaultValue
   let author = defaultValue
   let description = defaultValue
 
   try {
-    const parsed = YAML.parse(content)    
+    const parsed = YAML.parse(content)
     name = parsed.name ? sanitize(parsed.name) : defaultValue
     author = parsed.author ? sanitize(parsed.author) : defaultValue
-    description = parsed.description ? sanitize(parsed.description) : defaultValue
+    description = parsed.description
+      ? sanitize(parsed.description)
+      : defaultValue
   } catch (error) {
     // this happens in https://github.com/gaurav-nelson/github-action-markdown-link-check/blob/9de9db77de3b29b650d2e2e99f0ee290f435214b/action.yml#L9
     // because of invalid yaml
-    core.warning(`Error parsing action file [${filePath}] in repo [${repo}] with error: ${(error as Error)}`)
-    core.info(`The parsing error is informational, seaching for actions has continued`)    
+    core.warning(
+      `Error parsing action file [${filePath}] in repo [${repo}] with error: ${
+        error as Error
+      }`
+    )
+    core.info(
+      `The parsing error is informational, seaching for actions has continued`
+    )
   }
-  return { name, author, description }
+  return {name, author, description}
 }
 
 function sanitize(value: string) {
   return string.sanitize.keepSpace(value)
 }
+
+// Interface for a Dockerfile with actionable properties
+export interface DockerActionFiles {
+  [key: string]: string | undefined
+  name?: string
+  description?: string
+  // icon?: string
+  // color?: string
+
+  // Icon and color is needed for using dockerfiles as actions, but it's not used in the marketplace.
+}
+
+export const returnActionableDockerFiles = async (path: string) => {
```

Comment:
Rename method to match the other methods in this class (don't forget to update the call to this method as well
```suggestion
export const getActionableDockerFiles = async (path: string) => {
```

Reply:
Updated it in a later commit